libs/widget-lib/src/types.ts

```diff
@@ -1,5 +1,6 @@
 import type { SupportedChainId } from '@cowprotocol/cow-sdk'
 import { CowEventListeners, CowEventPayloadMap, CowEvents } from '@cowprotocol/events'
+import { CowSwapTheme } from '@cowprotocol/ui'
```

Comment:
To be more implicit we could set `type` after `import` it also helps to avoid including the whole library into another one

```
import type { CowSwapTheme } from '@cowprotocol/ui'
```